apps/jobs/views.py

```diff
@@ -228,3 +231,65 @@ def leaderboard(request, challenge_phase_split_id):
     paginator, result_page = paginated_queryset(distinct_sorted_leaderboard_data, request)
     response_data = result_page
     return paginator.get_paginated_response(response_data)
+
+
+@throttle_classes([UserRateThrottle])
+@api_view(['GET'])
+@permission_classes((permissions.IsAuthenticated, HasVerifiedEmail))
+@authentication_classes((ExpiringTokenAuthentication,))
+def get_remaining_submissions(request, challenge_phase_pk, challenge_pk):
+    challenge_model = get_model_object(Challenge)
+    challenge_model(challenge_pk)
+
+    challenge_phase_model = get_model_object(ChallengePhase)
+    challenge_phase = challenge_phase_model(challenge_phase_pk)
+
+    participant_team_pk = get_participant_team_id_of_user_for_a_challenge(
+        request.user, challenge_pk)
+
+    try:
```

Comment:
yeah, also write a comment stating that check for existence of participant team is deduced from above function. You can rephase the comment in your own words.